recipes/happly/all/conanfile.py

```diff
@@ -0,0 +1,37 @@
+from conans import ConanFile, tools
+import os
+import glob
+
+
+class HapplyConan(ConanFile):
+    name = "happly"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/nmwsharp/happly.git"
+    topics = ("conan", "happly", "ply", "3D")
+    license = "MIT"
+    description = "A C++ header-only parser for the PLY file format. Parse .ply happily!"
+    settings = "compiler"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return os.path.join(self.source_folder, "source_subfolder")
+
+    def validate(self):
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 11)
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = glob.glob(self.name + "-*/")[0]
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def package(self):
+        self.copy("LICENSE", src=self._source_subfolder, dst="licenses")
+        self.copy("happly.h", src=self._source_subfolder, dst=os.path.join("include", "happly"))
```

Comment:
No need since conan packages are unique

```suggestion
        self.copy("happly.h", src=self._source_subfolder, dst="include")
```